### PR TITLE
Need to specify the coverage file in travis.yml

### PR DESCRIPTION
**Pull request by elliotchance**
This change is [Reviewable](https://reviewable.io/reviews/codecov/example-go/10)

### CODE REVIEW

**Comment by elliotchance** [comment]
This fixes #9

File: .travis.yml [hunk 1]
```diff
@@ -11,4 +11,4 @@ script:
   - ./go.test.sh
 
 after_success:
-  - bash <(curl -s https://codecov.io/bash)
+  - include_cov=coverage.txt bash <(curl -s https://codecov.io/bash)
```
Comment (adiroiban) [inline .travis.yml:14]: why not use the `-f` argument which is documented in the output of the `--help` message?

the include_cov looks like a private variable which is not documented.
even in codecov.sh it has no comment attached to it... so no documentation

**Comment by adiroiban** [comment]
I have forked the repo and pushed the reports to my codecov.io account and it worked, even without `include_cov`

so I don't know if this is fixing any real issue.

but yes, it would help if the example code and the documentation could use an explicit coverage file

File: .travis.yml (hunk 1 above)
Comment (elliotchance) [reply .travis.yml:14]: You may be right. It took me a while to find a solution and this was the first one that worked for me: https://github.com/elliotchance/c2go/blob/master/.travis.yml#L47

I will try `-f` later and let you know.

**Pull request closed without merging**